puppet/services/opendaylight-api.yaml

```diff
@@ -32,6 +36,14 @@ parameters:
   OpenDaylightApiVirtualIP:
     type: string
     default: ''
+  OpendaylightPortBindingController:
+    description: Opendaylight port binding controller
+    type: string
+    default: 'pseudo-agentdb-binding'
+  OpendaylightHostconfURI:
```

Comment:
OpenDaylight, and I think it is better to include the extra 2 characters to spell out HostConfig